apps/src/redux.js

```diff
@@ -16,15 +16,17 @@ if (process.env.NODE_ENV !== "production") {
  * @param {!function} reducer
  * @return {Store} Configured Redux store, ready for use.
  */
-module.exports.createStore = function (reducer) {
+module.exports.createStore = function (reducer, initialState) {
```

Comment:
Added this because we had been using initialState to create our progress reduxStore. We're not anymore, but still worth having here.